screens/team-codes/JoinScreen.js

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from "react";
+import {
+  Image,
+  Text,
+  View,
+  StyleSheet,
+  TouchableOpacity,
+  TextInput
+} from "react-native";
+import { getTeamIdFromTeamCode, getTeam, joinTeam } from "../../utils/db/teams";
+import KitText from "../../components/KitText";
+import Colors from "../../constants/Colors";
+import FontStyles from "../../constants/FontStyles";
+import KitBackgroundScreen from "../../components/KitBackgroundScreen";
+import KitButtonSupreme from "../../components/KitButtonSupreme";
+
+export default function JoinScreen(props) {
+  const [textValue, setTextValue] = useState("");
+  const [teamName, setTeamName] = useState("");
+  const [teamFound, setTeamFound] = useState(false);
+
+  return (
+    <KitBackgroundScreen
+      onPressBack={() => {
+        props.navigation.navigate("Teams");
+      }}
+      title="Enter Code:"
+    >
+      <View style={styles.container}>
+        <TextInput
+          style={styles.textInput}
+          onChangeText={async text => {
+            setTextValue(text);
+            if (text.length == 6) {
```

Comment:
More a design question but should we somehow indicate on the page that we expect the team code to be 6 characters long, for a better user experience?

Reply:
yeah i'll come up with a list of things to let design know relating to create/join